cmd/svcat/README.md

```diff
@@ -204,11 +204,13 @@ Bindings:
 ## Unbind all applications from an instance
 ```
 svcat unbind -n test-ns ups-instance
+deleted ups-binding
```

Comment:
Would you please switch this to the following preferred format for displaying a command + its output?

```
```console
$ COMMAND
output
```
```